requirements/js_test.in

```diff
@@ -1,6 +1,4 @@
 # Requirements for Jasmine javascript testing framework
 
-jasmine==2.9.0
-cherrypy==17.3.0   # This dependency shouldn't be needed.
-                   # It should just be a dependency of jasmine, but without
-                   # this pinning it upgraded which broke jasmine tests.
+jasmine==3.1.0  # versions>3.1.0 cause test failures.
+cherrypy
```

Comment:
Given the comment above, can `cherrypy` be removed from this file now?

Reply:
I'll check it once on local by removing and if possible will remove it.